Backport new release notes structure to develop (#1316)
Avaiga/taipy-doc: docs/release-notes/2.0/index.md

---
title: Release Notes for version 2.0
---

> This is the list of changes of taipy version 2.0.

Reviewer: ```suggestion
These are the updates and changes introduced in Taipy version 2.0.
```
